app/hooks/useMix.tsx

```diff
@@ -14,21 +14,18 @@ type MixState = {
   error: { message: string }
   mixIndex: number
   userLikes: string[]
-  currentPage: number
   mix: C4Content[] | null
-  hasNextPage: boolean
   mixLimit: number
   mixIndexLimit: number // the number that will act as a trigger to get more content
```

Comment:
There's a notable performance difference with loading sites initially when this is set to 10 vs 100. Any opposition to setting it to something lower?